lib/lib_test.go

```diff
@@ -58,9 +59,13 @@ func TestReceivers(t *testing.T) {
 	}
 
 	node := n.(*p2p.QriNode)
-	reqs := Receivers(node)
-	if len(reqs) != 9 {
-		t.Errorf("unexpected number of receivers returned. expected: %d. got: %d\nhave you added/removed a receiver?", 9, len(reqs))
+	cfg := config.DefaultConfigForTesting()
+	// TODO (b5) - hack until tests have better instance-generation primitives
+	inst := &instance{node: node, cfg: cfg}
```

Comment:
From my perspective, I don't think this is a hack, I think this is the proper use.